siriuspy/siriuspy/clientconfigdb/configdb_client.py

```diff
@@ -170,6 +171,39 @@ def check_valid_value(self, value, config_type=None):
         config_type = self._process_config_type(config_type)
         return _templates.check_value(config_type, value)
 
+    @staticmethod
```

Comment:
I think this can be migrated to PVsConfig class

Reply:
I don't know. I don't usually use PVsConfig, I prefer to use this basic class.
maybe we could add a method in PVsConfig to compare the "self" with another config.
what do you think?

Reply:
yes, in the future we could do that here!

Reply:
ok!